src/main/java/book/store/online/service/impl/BookServiceImpl.java

```diff
@@ -1,5 +1,6 @@
 package book.store.online.service.impl;
 
+import book.store.online.exception.EntityNotFoundException;
```

Comment:
you should return and accept dtos in service layer

<img width="511" alt="image" src="https://github.com/ArtyomK75/online_book_store/assets/81781479/97600225-451d-4c1f-8054-a9aaa8365575">